src/main/java/Duke.java

```diff
@@ -271,9 +354,14 @@ private static void addEvent(String userInput) {
     private static void addDeadLine(String userInput) {
```

Comment:
Misleading naming convention. "DeadLine" can be interpreted as "Dead Line". Consider "Deadline" instead